products/neo4j.md

```diff
@@ -60,9 +60,9 @@ releases:
 
 -   releaseCycle: "4.4"
     eol: 2025-06-30
-    latest: "4.4.17"
+    latest: "4.4.19"
```

Comment:
Strange, https://github.com/neo4j/neo4j/releases/tag/4.4.18 should have been catched long ago. Maybe they forgot to push the tag ?